docs/data/data-grid/custom-columns/EditingWithDatePickers.tsx

```diff
@@ -102,18 +99,7 @@ function GridFilterDateInput(
       value={item.value ? new Date(item.value) : null}
       autoFocus
       label={apiRef.current.getLocaleText('filterPanelInputLabel')}
-      slotProps={{
-        textField: {
-          variant: 'standard',
-        },
-        inputAdornment: {
-          sx: {
-            '& .MuiButtonBase-root': {
-              marginRight: -1,
-            },
-          },
-        },
-      }}
+      slotProps={{ textField: { size: 'small' } }}
```

Comment:
Fix this visual inconsistency. 🙈 
![Screenshot 2024-12-23 at 18 26 27](https://github.com/user-attachments/assets/a3f92e7b-204a-4edc-abcc-557a4692145f)

Reply:
Thanks for catching this